.gitignore

```diff
@@ -61,10 +61,14 @@ docs/_build/
 
 # cython generated files
 PyMca5/PyMcaGraph/ctools/_ctools/cython/*.c
+PyMca5/PyMcaMath/mva/_cython_kmeans/*.c
```

Comment:
A little surprising that this is missing from upstream.

Reply:
My impression is that these get dynamically generated as needed, but I don't really have a good mental picture of why it's done that way.

Reply:
Skipping this for no, until I understand it better.